### PR TITLE
Fix service install bootstrap error 125 with retry logic

### PR DESCRIPTION
**Pull request by cboone**
## Summary

- Reorder `Install()` to write the plist before calling `bootout()`, ensuring the legacy bootout fallback always has the file on disk
- Add retry logic: if `launchctl bootstrap` fails with error 125 ("Domain does not support specified action"), bootout and retry once
- Call `launchctl enable` before retry to handle the edge case where the service was previously disabled
- Add `isAlreadyBootstrappedError()` helper (with unit tests) and `runBootstrap()` extraction

## Test plan

- [x] `make test` passes (all unit tests including new `TestIsAlreadyBootstrappedError`)
- [x] `make test-scrut` passes (65 CLI tests)
- [x] `make lint` passes (golangci-lint, markdownlint, actionlint)
- [ ] Manual: `snappy service install` from clean state (fresh install)
- [ ] Manual: `snappy service install` when service is already running (reinstall)
- [ ] Manual: `snappy service uninstall && snappy service install` (round-trip)

### CODE REVIEW

File: internal/service/launchd.go [hunk 1]
```diff
@@ -139,10 +136,39 @@ func Install(cfg PlistConfig) error {
...
+	outStr := strings.TrimSpace(string(out))
+	if !isAlreadyBootstrappedError(outStr) {
+		return fmt.Errorf("launchctl bootstrap: %s (%w)", outStr, err)
+	}
+
+	// Service is already registered in the domain. Try harder to unload it,
+	// then retry bootstrap once.
+	if bootoutErr := bootout(cfg.Label, plistPath); bootoutErr != nil {
+		return fmt.Errorf(
+			"launchctl bootstrap: service already loaded and bootout failed: %s (original: %s, %w)",
```
[the remark this answers is not in the document]
Comment (cboone) [reply internal/service/launchd.go]: Go's `fmt` package calls `Error()` on values implementing the `error` interface for both `%s` and `%v` verbs, so `%s` does not produce `%!s(<type>=<value>)` output with error values. However, `%v` is more idiomatic for errors, so updated to `%v` in e4b8e70. Updated Copilot instructions to note this Go behavior.
